### ISSUE TITLE
Use AlphaFold2 ptm for multimer chain-break prediction

### ISSUE THREAD

**Issue by sirius777coder** [sokrypton/ColabFold#661]
Hi,

Great works ! I want to use AlphaFold2 ptm model with chain breaks for homomers rather than AlphaFold_multimer_v3. Can I just turn the model type to alphafold2_ptm ? But I find it still returns ipTM score

**Comment by sirius777coder** [comment]
Thanks for reply!

**Comment by sirius777coder** [comment]
ello, I have a follow-up question regarding the ColabFold alphafold2_ptm model. I've noticed discrepancies in the pLDDT when running multimer models using vanilla AlphaFold2 PTM or ColabFold AlphaFold2 PTM weight. I'm curious about the cause of these differences. Could it be attributed to chain breaks or other factors?

**Comment by sirius777coder** [comment]
Hello, I have a follow-up question regarding the ColabFold alphafold2_ptm model. I've noticed discrepancies in the pLDDT when running multimer models using vanilla AlphaFold2 PTM or ColabFold AlphaFold2 PTM weight. I'm curious about the cause of these differences. Could it be attributed to chain breaks or other factors?

**Comment by milot-mirdita** [comment]
Are you testing with the the same input MSA?

**Comment by sirius777coder** [comment]
I used the single sequence MSA mode

**Comment by sokrypton** [comment]
The vanilla pTM model does not allow for input of multimers. We modified the code to allow for this.

How did you modify the code of the vanilla model?

**Comment by sirius777coder** [comment]
Thank you for your response! I added a 200-residue chain break and then modeled the complex sequences as a monomer. I think this is same with ColabFold alphafold2_ptm model with multimer?

**Comment by sirius777coder** [comment]
Oh, I've entered the MSA containing only the query sequence as a single line. This could be the reason why identical parameters are producing two distinct outcomes. Have you noticed any advantages of employing the block diagonal mode for AlphaFold2 PTM when predicting de novo designed sequences? I'm concerned that the AlphaFold2 Multimer parameters might bias the design results

**Comment by sokrypton** [comment]
Yes, from our experience it does significantly better to input as block diagonal

**Comment by sirius777coder** [comment]
Great! Thank you for your kind reply!
